LethalAPI.Core/LethalAPI.Core.csproj

```diff
@@ -9,6 +9,11 @@
         <DebugType>embedded</DebugType>
         <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
         <LangVersion>latest</LangVersion>
+        <RestoreAdditionalProjectSources>
+            https://api.nuget.org/v3/index.json;
+            https://nuget.bepinex.dev/v3/index.json;
+            https://nuget.samboy.dev/v3/index.json
```

Comment:
This source doesn't appear to be necessary?

Reply:
Removed 👍